### PR TITLE
Move all manipulation files in uuv_dave out of legacy

### PR DESCRIPTION
**Pull request by mabelzhang**
Partially addresses #142.
Moving all manipulation demo files out of `legacy/uuv_dave`.
Some of them had already been moved and renamed to other files, so just removing them here.

Files were moved for these tutorials
https://github.com/Field-Robotics-Lab/dave/wiki/Manipulator-Force-Feedback
https://github.com/Field-Robotics-Lab/dave/wiki/BOP-Panel-Manipulation-Mission

(Removed legacy files for these tutorials, which I realized had already been copied out of `legacy` and renamed. Tutorials have been updated.
https://github.com/Field-Robotics-Lab/dave/wiki/Swapping-out-the-Oberon7-arm-with-another-manipulator/
https://github.com/Field-Robotics-Lab/dave/wiki/Teleop-Bar-Retrieval
)

TODO(mabelzhang) after this PR is merged: Update tutorial launch commands as follows.
- [x] `roslaunch uuv_dave uuv_manipulator_spring_demo.launch` -> `roslaunch dave_demo_launch dave_manipulator_spring_demo.launch`
- [x] `roslaunch uuv_dave plot_manipulator.launch` -> `roslaunch dave_demo_launch dave_manipulator_spring_plot.launch`
- [x] `roslaunch uuv_dave valve_manipulation_demo.launch` -> `roslaunch dave_demo_launch dave_valve_manipulation_demo.launch`

### CODE REVIEW

**Comment by M1chaelM** [comment]
@mabelzhang Do these tutorials work for you? I'm getting a crash because of a syntax error in `gripper.py` from `uuv_manipulators_kinematics`. It's an easy fix; just trying to figure out if I'm missing some change that handles it already.

**Review by M1chaelM — APPROVED** [review]
I'm approving because there are no longer files related to manipulation tutorials in the legacy folder, which is the goal of this PR.

As I mentioned in the comments, the actual manipulation seems to be broken right now because of python errors in `uuv_manipulators.` These are easy to fix so I'll put in a PR for them.

**Comment by mabelzhang** [comment]
Thank you! Approved https://github.com/Field-Robotics-Lab/uuv_manipulators/pull/6
Will merge this once that's merged.

**Pull request merged**
